resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/PVTestsSuite.scala

```diff
@@ -111,15 +109,28 @@ private[spark] trait PVTestsSuite { k8sSuite: KubernetesSuite =>
       .storageClasses()
       .withName(STORAGE_NAME)
       .delete()
+
+    mountProcess.foreach(_.destroy())
+
   }
 
   private def checkPVs(pod: Pod, file: String) = {
+    implicit val podName: String = pod.getMetadata.getName
+    implicit val components: KubernetesTestComponents = kubernetesTestComponents
     Eventually.eventually(TIMEOUT, INTERVAL) {
-      implicit val podName: String = pod.getMetadata.getName
-      implicit val components: KubernetesTestComponents = kubernetesTestComponents
-      val contents = Utils.executeCommand("cat", s"$CONTAINER_MOUNT_PATH/$file")
-      assert(contents.toString.trim.equals(FILE_CONTENTS))
+      Utils.executeCommand("ls", s"$CONTAINER_MOUNT_PATH/")
```

Comment:
Do we need to check this additionally before checking `cat` the file content?

Reply:
This is to improve the debugging. We can see if the directory exists first and then check the file contents and have the failures be separate. This makes it easier to see if the PV mount failed or if the bind mount into minikube failed.

Reply:
Got it~